src/create-root.js

```diff
@@ -11,6 +11,8 @@ import { mount } from './diff/mount';
 import { patch } from './diff/patch';
 import { createInternal } from './tree';
 
+let rootId = 0;
```

Comment:
We might want to introduce some randomness here in case there are multiple completely separate Preact installations running, i.e. 2 widgets as those will be two roots with a resetting `_domDepth`

Reply:
FYI: In devtools I'm using this to sort of namespace the ids of different preact installations

```js
const namespace = Math.floor(Math.random() * 2 ** 32);
```

https://github.com/preactjs/preact-devtools/blob/15762ba29ea1cfeca6964bb09dadf4dc625d2bde/src/adapter/hook.ts#L211

Reply:
Randomness would break hydration with SSR, though, no?

Reply:
Right, good point about the randomness!